common/lib/xmodule/xmodule/split_test_module.py

```diff
@@ -695,3 +695,11 @@ def _create_vertical_for_group(self, group, user_id):
         )
         self.children.append(dest_usage_key)  # pylint: disable=no-member
         self.group_id_to_child[unicode(group.id)] = dest_usage_key
+
+    @property
+    def tooltip_content(self):
+        if self._xmodule is None:
+            return ''
+        if self._xmodule.child is None:
+            return ''
+        return self._xmodule.child.tooltip_content if self._xmodule.child.tooltip_content else ''
```

Comment:
I tried that and discovered that when this property is added to the module, it also becomes necessary to override the property on the descriptor, or the descriptor will use the default implementation and display its own display_name (i.e. "Split test") in the tooltip when the split test is the top level object in the sequence item.